boost.sh

```diff
@@ -1423,7 +1451,7 @@ fi
 # -DBOOST_SP_USE_SPINLOCK
 # -stdlib=libc++
 
-CPPSTD="-std=c++1z"
+CPPSTD="-std=c++14"
```

Comment:
We use c++17

Reply:
What is the minimum libstdc++ version required? In general c++14 is a stable iteration, while 17 has a lot of new features... I am all up to stabler rather than newer. Not a real problem in case the libstdc++ supports both...

Reply:
Actually @dipankadas and I had problems with and `libstdc++.so.7`, why do you need C++17 for, specially to compile a dependency? Couldn't you just compile third party libraries like boost with C++14 and enable C++17 on the compilation units you really use the desired features?

Reply:
We could try, since we dont use binary libraries from boost it shouldnt affect us.

Reply:
I checked and we the docker image has `libstdc++.so.6.0.28`. Our rtc-cpp circle-ci image has `libstdc++.so.6.0.22`. Here is a list of gcc and their respective libstdc++.so minimum required version: https://gcc.gnu.org/onlinedocs/libstdc++/manual/abi.html . I've also compiled some stuff with gcc-8 and `-std=c++17` on  and it ran OK with `libstdc++.so.6.0.22`. So we should be fine either way.